src/dart/pubspec.yaml

```diff
@@ -0,0 +1,17 @@
+name: pose
+description: Dart library for loading, viewing, augmenting, and handling .pose files
+version: 1.1.5
+repository: https://github.com/bipinkrish/pose
```

Comment:
should we move this away? if you want to keep it under your repository, maybe we should create a git sub-module that just links to your repo?

Reply:
That is for [pub.dev](pub.dev) to link a repository.

Reply:
> maybe we should create a git sub-module that just links to your repo?

Your call, either is ok for me.